README.md

```diff
@@ -1,15 +1,50 @@
 # Antiplagiarism
+Antiplagiarism is a Visual Studio extension that can compare the C# code in order to find similar methods. The tool can be used to compare the source code with itself
+in order to collect data for refactoring or with a custom code provided via path to folder/solution file/project file. 
```

Comment:
From my point of view this sentence is too complicated. I think words "folder/solution file/project" should not be used.

Reply:
Would you prefer something like 
"The tool can be used to compare the source code with itself in order to collect data for refactoring or with a provided custom code"?

Reply:
I think it is better to rephrase to something like "The tool can be used to compare the source code with itself or with a provided custom code in order to collect data for refactoring"

Reply:
@metallmonster done